raw_data/preprocessed/africa/zimbabwe_english.txt

```diff
@@ -0,0 +1,464 @@
+
+Amendment of First Schedule
+(141) [1] The Minister may, by statutory instrument, amend the First Schedule by adding any item to it, deleting any item from it or altering any item in it. [2] In the exercise of his powers under subsection [1], the Minister may define any activity as a project by reference to its magnitude.
+
+Amendment of Acts
+(142) The Act specified in each Part of the Sixth Schedule is amended to the extent set out in that Part.
+
+Repeal of Cap 20:13 and savings
+(143) [1] In this section: “conservation committee” means a conservation committee established for an intensive conservation area in terms of the repealed Act; “repealed Act” means the Natural Resources Act [Chapter 20:13]. [2] Subject to this section the Natural Resources Act [Chapter 20:13] is repealed. [3] Any regulations, rules, by-laws, notices, orders or awards which, immediately before the fixed date, were in force under the repealed Act shall remain in force as if they had been made or granted under this Act. [4] Any matter or thing which was lawfully made, done or commenced under the repealed Act and which, immediately before the fixed date, had effect or was capable of acquiring effect shall, subject to this Act, continue to have or to be capable of acquiring force, as the case may be, and shall be deemed to have been made, done or commenced under this Act. [5] The Minister shall issue written directions to every conservation committee established in terms of the repealed Act specifying the terms and conditions under which and the procedures by which a rural district council shall succeed to the conservation committee's assets and liabilities.
+
+Repeal of Caps. 20:03, 15:05 and 19:07
+(144) The Air Pollution Prevention Act [Chapter 20:03], the Hazardous Substances and Articles Act [Chapter 15:05] and the Noxious Weeds Act [Chapter 19:07] are repealed. [This section is not yet in force, as at 31st May, 2004, by virtue of S.I. 103 of 2003—Editor.]
```

Comment:
Please remove everything after `repealed.`. For that matter, please remove any of the editor's comments that I missed elsewhere in this legislation.